wordle-web/pages/wordEditor.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <v-container>
+    <v-card class="text-center">
+      <v-card-title>Word Editor</v-card-title>
+      <v-card-text>
+        <v-form @submit.prevent="addWord">
+          <v-text-field
+            v-model="newWord"
+            label="Add a new word"
+            required
+          ></v-text-field>
+          <v-checkbox
+            v-model="isCommon"
+            label="Common word"
+          ></v-checkbox>
+          <v-btn type="submit" color="primary">Add Word</v-btn>
+        </v-form>
+        <v-divider class="my-5"></v-divider>
+        <v-text-field
+          v-model="searchQuery"
+          label="Search words"
+        ></v-text-field>
+        <v-checkbox
+          v-model="filterCommon"
+          label="Show only common words"
+        ></v-checkbox>
+        <v-list>
+          <v-list-item
+            v-for="(word, index) in paginatedWords"
+            :key="index"
+          >
```

Comment:
you might want to show the common word value in the list

Reply:
It is being shown now thank you!